lib/utils/LanguageHelper.dart

```diff
@@ -590,4 +590,42 @@ class LanguageHelper {
     }
     return OpenFoodFactsLanguage.UNDEFINED;
   }
+
+  static Map<String, String>? toJsonStringMap(
+      Map<OpenFoodFactsLanguage, String>? map) {
+    return toJsonMap(map);
+  }
+
+  static Map<OpenFoodFactsLanguage, String>? fromJsonStringMap(
```

Comment:
I'm not sure I understand the point of creating several methods that do the same thing - except for a simple cast of data. In that case please comment each method.

Reply:
Yes, they're for casts.
To be precise, they are for the `fromJson` and `toJson` properties of the `@JsonKey` annotation.
The annotation does not support generic functions, otherwise those extra not-generic functions wouldn't be needed.

I'll add comments :)